dev-docs/modules/userId.md

```diff
@@ -376,42 +376,62 @@ pbjs.setConfig({
 
 ### LiveIntent ID
 
-LiveIntent offers audience resolution by leveraging our next-generation identity solutions. The LiveIntent identity graph is built around a people-based set of data that is authenticated daily through active engagements with email newsletters and media across the web. The LiveIntent ID is a user identifier tied to an active, anonymized email hash in our graph that functions in cookie-challenged environments like mobile browsers.
+LiveIntent offers audience resolution by leveraging our next-generation identity solutions. The LiveIntent identity graph is built around a people-based set of data that is authenticated daily through active engagements with email newsletters and media across the web. The LiveIntent ID is a user identifier tied to an active, anonymized email hash in our graph and functions in cookie-challenged environments and browsers.
```

Comment:
we are trying to stay away from saying "email hash" this caused a lot of negative publicity for the IAB when they used the term.  We prefer "encrypted email", its up to you.